README.md

```diff
@@ -142,7 +142,10 @@ This test suite expects following environment variables set:
  - `STORAGE_API_URL` - URL of Keboola Storage API (https://connection.keboola.com/)
  - `STORAGE_API_TOKEN` and `STORAGE_API_LINKING_TOKEN` - Storage API token associated to user (Admin master token) with all permissions. Project must have assigned `snowflake` and `redshift` backend. STORAGE_API_TOKEN and STORAGE_API_LINKING_TOKEN have to be tokens to different project in same organization.
  - `STORAGE_API_MAINTENANCE_URL` - URL for maintenance testing (https://maintenance-testing.keboola.com/)
-
+ - `STORAGE_API_PROJECT_IDS_IN_ORGANIZATION` - IDs of projects in organization, where one must be ID associated with `STORAGE_API_TOKEN`
```

Comment:
Sorry @pivnicek, I forgot add some variable descriptions. Can you please check this too? Thanks

Reply:
for readme it's fine.  but fyi, `the oraganization`, and `where one of them must be the ID ...` :-)